### PR TITLE
Fix/mock no rmq

### PR DESCRIPTION
**Pull request by andre-merzky**
Same as #630 but targeting the `feature/no_rmq_2` branch.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +0,0 @@
-
-radical.utils>=1.12
-radical.pilot>=1.18
-packaging
-requests>=2.4.0
```
Comment (mtitov) [inline requirements.txt:5]: requirements files got deleted - that was accidentally, right?

File: examples/misc/scale.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import time
+
+from radical.entk import Pipeline, Stage, Task, AppManager
+
+host = os.environ.get('RMQ_HOSTNAME', 'localhost')
+port = os.environ.get('RMQ_PORT',     5672)
```
Comment (mtitov) [inline examples/misc/scale.py:10]: these lines could be removed

File: examples/misc/scale.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+                s.add_tasks(t)
+
+            p.add_stages(s)
+
+        pipelines.add(p)
+
+
+    appman = AppManager(autoterminate=True, rts='mock')
+    appman.resource_desc = {'resource': 'local.localhost',
```
Comment (mtitov) [inline examples/misc/scale.py:46]: should we use another resource - `local.localhost_test`? which will not create a new VE, and will reuse the current one

File: requirements-tests.txt [hunk 4]
```diff
@@ -2,15 +2,9 @@
...
-pytest-xdist
 pylint
 flake8
-flake8-per-file-ignores
-psutil
-mock==2.0.0.
-hypothesis
 coverage
-codecov
-timeout-decorator
```
Comment (mtitov) [inline requirements-tests.txt:15]: were that removed packages excessive?

Comment (andre-merzky) [reply requirements-tests.txt]: Thanks for catching this!  I messed something up with the requirements.  Well, it's a good opportunity to test if they are needed actually :-)  I re-added those which are actually used.

File: examples/misc/scale.py (hunk 2 above)
Comment (andre-merzky) [reply examples/misc/scale.py]: Done, thanks - and some more too...

File: examples/misc/scale.py (hunk 3 above)
Comment (andre-merzky) [reply examples/misc/scale.py]: Right, done.

File: requirements.txt (hunk 1 above)
Comment (andre-merzky) [reply requirements.txt]: Thanks, resolved.

**Review by mtitov — APPROVED** [review]
LGTM!

**Pull request merged**